migrations/Version20171023043854.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace ChoreDo\Migrations;
```

Comment:
Namespaces should be uniformly camel or lowercase. We've got a bit of each. Maybe do all LC for this PR and then make a separate PR to CC it?